lib/debug/color.rb

```diff
@@ -0,0 +1,29 @@
+require "irb/color_printer"
+
+module DEBUGGER__
+  module Color
+    def colorize str, color
+      if CONFIG[:use_colorize]
+        IRB::Color.colorize str, color
+      else
+        str
+      end
+    end
+
+    def colored_inspect(obj)
+      if CONFIG[:use_colorize]
+        IRB::ColorPrinter.pp(obj, "")
+      else
+        obj.pretty_inspect
+      end
+    end
+
+    def colorize_cyan(str)
+      colorize(str, [:CYAN, :BOLD])
+    end
+
+    def colorize_blue(str)
```

Comment:
not directly related, but I plan to name coloring methods after the type of data they're used on. for example, this could be `colorize_sig`.